django_app/redbox_app/redbox_core/models.py

```diff
@@ -94,6 +94,25 @@ class Profession(models.TextChoices):
         VET = "VET", _("Veterinary")
         OT = "OT", _("Other")
 
+    class AIExperienceLevel(models.TextChoices):
```

Comment:
How sure are we that we're going to keep these categories and labels? It'll be annoying to update them in the future.

Reply:
I did talk to Charlotte about this. She said that the categories would be fairly stable, so I decided on an enumeration over a foreign key. If it turns out they do change often, we might reverse that decision in future.